Hunting Queries/BehaviorAnalytics/Anomalous Defensive Mechanism Modification.yaml

```diff
@@ -0,0 +1,18 @@
+eid: fcb704ae-ac17-400a-9ed9-3c46bd0a3960
+name: Anomalous Defensive Mechanism Modification
+description: |
+    'Adversaries may disable security tools to avoid possible detection of their tools and activities. DarkComet, for example, can disable Security Center functions like anti-virus. The query below generates an output of all users performing a "delete" operation regarding a security policy, where one or more features of the activitiy deviates from the user, his peers or the tenant profile.'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+        - BehaviorAnalytics
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  - T1562
+query: |
+    let operations = dynamic(['Remove database vulnerability assessment rule baseline']);
+    BehaviorAnalytics
+    | where ActionType in(operations)
+    | where ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done